src/MockBot.Api/appsettings.json

```diff
@@ -7,8 +7,8 @@
   },
   "AllowedHosts": "*",
   "DmrServiceSettings": {
-    "DmrApiUri": "https://tablelogapi.azurewebsites.net/app/frommockbot",
-    "DmrRequestProcessIntervalMs": 5000
+    "CentOpsUri": "http://127.0.0.1:8080",
+    "CentOpsApiKey": "<generate a key>"
   },
```

Comment:
Is `DmrRequestProcessIntervalMs` still required? There is still config for processing batched up requests right?  I think the CentOps related properties should go under a new section (`CentOpsSettings` or something).  Please correct me if I'm mistaken!

Reply:
This was the default - but I wanted to leave this in to allow the polling intervals to change via configuration at a later date.
I've done the same serialization tricks to avoid a mass rename of settings.

Reply:
We discussed and this is all good 👍🏽